### ISSUE TITLE
Ayla

### ISSUE THREAD

**Issue by ban-tantale** [mtgred/netrunner#3501]
Seen three times in the last 24h.  After Ayla selects her NVRAM, the corp sees only the back of their cards.  This bug seems to occur a bit randomly (I had at least two successful games between these bugs).

**Comment by jwarwick** [comment]
Maybe related to #3175, though this could be a separate `Ayla` issue

**Comment by ban-tantale** [comment]
Yes.  Compared to #3502, we did not try to leave/refresh.  The fact that it happened 3 times in about 6 games with Ayla, and the fact that Ayla has a special procedure at the start, are what made me suspect Ayla.
